src/Decompiler/Analysis/IndirectCallRewriter.cs

```diff
@@ -54,11 +58,13 @@ public class IndirectCallRewriter
             this.expander = new IndirectCallExpander(ssa);
             this.ssaIdTransformer = new SsaIdentifierTransformer(ssa);
             this.eventListener = eventListener;
+            this.emiter = new ExpressionEmitter();
```

Comment:
It's spelled `emitter`; I'm considering switching from `emitter` to `m` everywhere, to be consistent with over 1000 unit tests that use `m`. Any thoughts?